packages/api/src/getRollups.ts

```diff
@@ -0,0 +1,9 @@
+import { Rollup } from "@blobscan/db";
```

Comment:
These changes were removed in the rollup filter refactor. This should be an error in managing conflicts after many merges